InformationScripting/src/InformationScriptingPlugin.cpp

```diff
@@ -28,15 +28,23 @@
 #include "SelfTest/src/SelfTestSuite.h"
 
 #include "OOInteraction/src/handlers/HStatementItemList.h"
+#include "OOModel/src/declarations/Method.h"
+#include "OOModel/src/declarations/Class.h"
 
 #include "commands/CScript.h"
 #include "helpers/BoostPythonHelpers.h"
+#include "sources/AstSource.h"
+
+#include "visitors/AllNodesOfType.h"
 
 namespace InformationScripting {
 
 bool InformationScriptingPlugin::initialize(Core::EnvisionManager&)
 {
 	BoostPythonHelpers::initializeQStringConverters();
+	AstSource::init();
+	AllNodesOfType<OOModel::Method>::init();
```

Comment:
Is this supposed to be an initialization of a filter? Why do we need to initialize here? Something is fishy, we shouldn't have to initialize this for each and every type. We should find a way to make this unnecessary.